app/controllers/invite_controller.rb

```diff
@@ -0,0 +1,26 @@
+class InviteController < ApplicationController
+  def new
+
+  end
+
+  def create
+    sender_name = "#{current_user.first_name} #{current_user.last_name}"
+    github_service = GithubService.new
+    inform_info = github_service.get_user_info(invite_params[:github_handle], current_user.token)
+    if inform_info[:email]
```

Comment:
We can refactor this to it's own method to make this create action a little bit more clear